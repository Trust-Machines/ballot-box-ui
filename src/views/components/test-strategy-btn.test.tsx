import {render} from '@testing-library/react';

import TestStrategyBtn from './test-strategy-btn';

test('1 Default render', () => {
    const view = render(<TestStrategyBtn strategy="stx-balance" network="mainnet" strategyOptions={{}}/>);
    expect(view.container).toMatchSnapshot()
});

test('2 Empty strategy', () => {
    const view = render(<TestStrategyBtn strategy="empty" network="mainnet" strategyOptions={{}}/>);
    expect(view.container).toMatchSnapshot()
});
