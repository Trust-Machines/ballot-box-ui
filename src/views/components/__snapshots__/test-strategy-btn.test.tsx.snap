// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`1 Default render 1`] = `
<div>
  <button
    class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedInfo MuiButton-sizeSmall MuiButton-outlinedSizeSmall css-1kmv2jn-MuiButtonBase-root-MuiButton-root"
    tabindex="0"
    type="button"
  >
    Test Strategy
    <span
      class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
    />
  </button>
</div>
`;

exports[`2 Empty strategy 1`] = `<div />`;
